scripts/generator-adapter/generators/app/index.ts

```diff
@@ -0,0 +1,298 @@
+import * as Generator from 'yeoman-generator'
+interface Transport {
+  type: string
+  name: string
+}
+interface Endpoint {
+  endpointName: string
+  normalizedEndpointName: string
+  transports: Transport[],
+  endpointAliases: string
+  normalizedEndpointNameCap: string
+}
+
+module.exports = class extends Generator {
+  props: {
+    frameworkVersion: string
+    adapterName: string,
+    endpoints: Record<string, Endpoint>,
+    endpointNames: string,
+    defaultEndpoint: Endpoint
+  }
+
+  constructor(args, opts) {
+    super(args, opts);
+    this.argument("rootPath", { type: String, required: false, default: './', description: 'Root path where new External Adapter will be created'});
+  }
+  async prompting() {
+    let { adapterName } = await this.prompt( {
+      type: "input",
+      name: "adapterName",
+      message: "What is the name of adapter?:",
+      default: "example-adapter"
+    })
+
+    adapterName = this._normalizeStringInput(adapterName)
+    if (adapterName === '') {
+      throw new Error('Adapter name cannot be empty')
+    }
+
+    let { endpointCount } = await this.prompt({
+      type: "input",
+      name: "endpointCount",
+      message: "How many endpoints does adapter have?:",
+      default: "1"
+    })
+
+    endpointCount = parseInt(endpointCount)
+
+    if (isNaN(endpointCount) || endpointCount <= 0) {
+      throw new Error('Adapter should have at least one endpoint')
+    }
+
+    const endpoints: Record<string, Endpoint> = {}
+    for (let i = 0; i < endpointCount; i++) {
+      let { endpointName } = await this.prompt<{endpointName: string}>({
+        type: "input",
+        name: "endpointName",
+        message: `What is the name of endpoint #${i+1}:`,
+        default: "price"
+      })
+
+      endpointName = this._normalizeStringInput(endpointName)
+
+      if (endpointName === '') {
+        throw new Error('Endpoint name cannot be empty')
+      }
+
+      const existingEndpoint = Object.values(endpoints).find(e => e.endpointName === endpointName)
+      if (existingEndpoint) {
+        throw new Error(`There is already configured endpoint with name '${endpointName}'`)
+      }
+
+      const { endpointAliasesAnswer } = await this.prompt<{endpointAliasesAnswer: string}>({
+        type: "input",
+        name: "endpointAliasesAnswer",
+        message: `Comma separated aliases for endpoint '${endpointName}':`,
+        default: "empty"
+      })
+      let endpointAliases
+
+      if (endpointAliasesAnswer === 'empty') {
+        endpointAliases = []
+      }else {
+        endpointAliases = endpointAliasesAnswer.split(',')
+      }
```

Comment:
Adds support for spaces just in case

```suggestion
      const endpointAliases = endpointAliasesAnswer === 'empty' ? [] : endpointAliasesAnswer.replace(/ /g, '').split(',')
```